### ISSUE TITLE
Reimplement superscript join with subscript into one piece

### ISSUE THREAD

**Issue by KrasnayaPloshchad** [alerque/libertinus#410]
Although there are some TeX engines allowed to make superscript join with subscript into one block (for example, chemical formula), there are some word processors still unable to reproduce, such as OpenOffice, AbiWord. And LibreOffice has so limited math support. So I suggest to implement it again, but not simply add kerning behaviour between these characters, instead, considering make use of ZWJ to do. This feature shall not be added in Libertinus Math font.

**Comment by alerque** [comment]
For more context see the original issue #355 and discussion and current resolution in PR #383 (and followup 8d17f44).

**Comment by KrasnayaPloshchad** [comment]
That’s why I proposed using ZWJ.

**Comment by alerque** [comment]
Yup yup, and thanks for the proposal. I just linked it so I or others could more easily find the preceding discussions from here.

For the record I'd probably be willing to accept a PR that contributes this, but given that I don't personally have any need for it, the whole thing is a bit of a hack, and the use case supporting non-compliant software for something it wasn't designed for is a bit dubious I probably won't be sinking a bunch of my own time into tackling this myself any time soon.

**Comment by KrasnayaPloshchad** [comment]
EB Garamond still have this feature
